### PR TITLE
Update to gh-pages

### PR DESCRIPTION
**Pull request by dhill**
I have updated the gh-pages branch to provide some branding and overview of the PSM project.  I have incorporated the previous SGC documentation into the site as well as added contact information, an overview of the project, purpose, and frequently asked questions.

Please let me know if you have any questions or concerns.

Thanks!
Dave

### CODE REVIEW

**Comment by cecilia-donnelly** [comment]
Note to all: please do not merge until I give the +1.  We're waiting on approval.

File: contact.html [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
+layout: default
 title: Contact
 permalink: /contact/
 ---
-<p>Since the Poplin team is spread across multiple organizations, we communicate using a few different channels:</p>
+<h1>Contact</h1>
 <ul>
-  <li>If you have general questions about the Provider Screening Module, please feel free to email us at <a href="mailto:info@projectpoplin.org">info@projectpoplin.org</a>.
-  <li>We send out periodic announcements on the <a href="https://twitter.com/project_poplin">Project Poplin twitter account</a>
+  <li>Have general questions about the Provider Screening Module? <br/>Join the discussion at <a href="mailto:psm-dev@googlegroups.com">psm-dev@googlegroups.com</a>.
```
Comment (cecilia-donnelly) [inline contact.html]: +1

File: documentation.html [hunk 2]
```diff
@@ -18,20 +18,18 @@ <h1>Documentation</h1>
...
 
-<h2>User guide</h2>
+<h2>User Manual</h2>
 
 <p>
-  You can read the <a href="/userdocs/html/index.html">user manual</a>
-  for PSM users (state agency staff, system administrators,
-  medical providers, and agents serving providers).
+  The user manual provides details on how different types of users (state agency staff, system administrators, medical providers, and agents serving providers) can use the Provider Screening Module.  It is available in 
+  several formats:
```
Comment (cecilia-donnelly) [inline documentation.html]: Please wrap this text.

File: faqs.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+## Who could benefit from using or joining the Provider Screening Module?
+
+States:
+* Provide input and feedback to influence development 
+* Early involvement enables easier integration with existing MMIS
+
+Vendors:
+* Shape open source API discussions
+* Contribute to development: add new features
+* Leverage for your product line
```
Comment (cecilia-donnelly) [inline faqs.md]: Leverage *code*, maybe?

File: faqs.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+Death Master File (DMF), and the Excluded Parties List System (EPLS).  The applicant's provider
+license is also checked.  Screening is necessary to verify that providers are qualified to
+care for Medicaid patients.
+
+## Where did the Provider Screening Module project come from?
+
+The Provider Screening Module started as part of a crowd-sourced challenge back in 2012.  The
+resulting implementation was evaluated by Minnesota and South Carolina, but it was not
+integrated into their systems.  From 2016 to 2017, CMS, with help from MITRE and SGC, resumed
+the project, performed a gap analysis on the existing implementation and developed a minimum
```
Comment (cecilia-donnelly) [inline faqs.md]: Oxford comma?

File: faqs.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+resulting implementation was evaluated by Minnesota and South Carolina, but it was not
+integrated into their systems.  From 2016 to 2017, CMS, with help from MITRE and SGC, resumed
+the project, performed a gap analysis on the existing implementation and developed a minimum 
+viable pilot.
+
+## How can I contribute to the Provider Screening Module?
+
+There are a few ways you can contribute to the Provider Screening Module:
+
+- Email general questions or comments to our [discussion list](mailto:info@projectpoplin.org)
```
Comment (cecilia-donnelly) [inline faqs.md]: Change to `psm-dev` list -- psm-dev AT googlegroups DOT com.

File: userdocs/html/system-admin-help.html [hunk 6]
```diff
@@ -46,7 +46,7 @@ <h3>Navigation</h3>
       <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
         <div class="sphinxsidebarwrapper">
             <p class="logo"><a href="index.html">
-              <img class="logo" src="_static/psm-logo-small.png" alt="Logo"/>
+              <img class="logo" src="static/psm-logo-small.png" alt="Logo"/>
```
Comment (cecilia-donnelly) [inline userdocs/html/system-admin-help.html]: Please squash these `_static` to `static` changes, or pull them out into a separate commit.

File: contact.html [hunk 7]
```diff
@@ -5,6 +5,14 @@
...
 <ul>
-  <li>Have general questions about the Provider Screening Module? <br/>Join the discussion at <a href="mailto:psm-dev@googlegroups.com">psm-dev@googlegroups.com</a>.
+  <li>
+  	Have general questions about the Provider Screening Module? <br />
+  	Join the discussion at <a href="mailto:psm-dev@googlegroups.com">psm-dev@googlegroups.com</a>.
+  </li>
+  <li>
+  	If you are a State, join our bi-weekly call - 
+  	<a href="mailto:psm-dev@googlegroups.com?subject=Bi-weekly%20Meeting%20Inquiry%20for%20PSM">contact the administrator</a> 
+  	for an invite.
```
Comment (cecilia-donnelly) [inline contact.html]: :+1:

File: faqs.md [hunk 8]
```diff
@@ -43,16 +43,18 @@ care for Medicaid patients.
...
-integrated into their systems.  From 2016 to 2017, CMS, with help from MITRE and SGC, resumed
-the project, performed a gap analysis on the existing implementation and developed a minimum 
-viable pilot.
+The Provider Screening Module started as part of a crowd-sourced challenge through 
+[Topcoder](https://www.topcoder.com/) back in 2012.  The resulting implementation was evaluated 
+by Minnesota and South Carolina, but was not adopted.  From 2016 to 2017, the 
+[Centers for Medicare and Medicaid Services (CMS)](https://www.cms.gov), with help from 
+[MITRE](https://www.mitre.org) and [Solution Guidance Corporation](http://www.solutionguidance.com), 
+resumed the project, performed a gap analysis on the existing implementation and developed a 
+minimum viable pilot.
```
Comment (cecilia-donnelly) [inline faqs.md]: Thanks for the links!

**Comment by jasonaowen** [comment]
The Checkstyle failure is unrelated. I've blacklisted the `gh-pages` target branch in Jenkins, so Checkstyle and the other integrations should not be run on future pull requests (including this one) that change the `gh-pages` branch. Sorry for the noise!

**Comment by cecilia-donnelly** [comment]
@dhill, sounds like we are good to go from an approval standpoint.  Can you respond to my code comments / make the needed changes?

File: _includes/head.html [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:300,300i,700" rel="stylesheet">
+  <script src="https://use.fontawesome.com/5a84c357fc.js"></script>
+  <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    ga('create', 'UA-104679487-1', 'auto');
+    ga('send', 'pageview');
```
[the remark this answers is not in the document]
Comment (dhill) [reply _includes/head.html]: Good catch - I think we should have a separate Google Analytics account setup so that we can see where the interest is coming from.  I wouldn't wait to add it.

File: _config.yml [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+# Site settings
+title: Open Source Provider Screening Module (PSM)
+# email: your-email@domain.com
+description: > # this means to ignore newlines until "baseurl:"
+  The PSM project seeks to develop a cohesive, free, open-source
+  provider screening module for states to use in developing a
+  modern, modular Medicaid IT system.
+baseurl: "" # the subpath of your site, e.g. /blog
+url: "http://providerscreeningmodule.org" # the base hostname & protocol for your site
```
[the remark this answers is not in the document]
Comment (dhill) [reply _config.yml]: Yes.

File: _includes/header.html [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+    <a class="site-title" href="{{ site.baseurl }}/"><img src='/images/poplin1x.png' alt='Small Poplin Logo' height='40px' /></a>
+
+    <nav class="site-nav">
+      <a href="#" class="menu-icon">
+        <svg viewBox="0 0 18 15">
+          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
+          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
+          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
+        </svg>
+      </a>
```
[the remark this answers is not in the document]
Comment (dhill) [reply _includes/header.html]: Left over from something else.  Removed.

File: _layouts/post.html [hunk 12]
```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+---
+<article class="post" itemscope itemtype="http://schema.org/BlogPosting">
+
+  <header class="post-header">
+    <h1 class="post-title" itemprop="name headline">{{ page.title }}</h1>
+    <p class="post-meta">
+    	<time datetime="{{ page.date | date_to_xmlschema }}" itemprop="datePublished">{{ page.date | date: "%b %-d, %Y" }}</time>{% if page.author %} • <span itemprop="author" itemscope itemtype="http://schema.org/Person"><span itemprop="name">{{ page.author }}</span></span>{% endif %}
```
[the remark this answers is not in the document]
Comment (dhill) [reply _layouts/post.html]: Done.

File: _sass/_base.scss [hunk 13]
```diff
@@ -0,0 +1,211 @@
...
+        display: inline-block;
+        width: 16px;
+        height: 16px;
+        vertical-align: middle;
+
+        path {
+            fill: $grey-color;
+        }
+    }
+}
```
[the remark this answers is not in the document]
Comment (dhill) [reply _sass/_base.scss]: This comes from MITRE.

**Comment by cecilia-donnelly** [comment]
Hey @dhill, I see a couple comments where you say you've changed something, but I don't see a corresponding commit.  They'll show up once you push.  (No rush, just wanted to be clear.)

File: userdocs/html/system-admin-help.html (hunk 6 above)
Comment (dhill) [reply userdocs/html/system-admin-help.html]: I changed this because it broke under the new brochure layout.
